### ISSUE TITLE
Client web app deploying issue

### ISSUE THREAD

**Issue by ME-ON1** [kelseyhightower/nocode#4313]
But i don't know how to integrate this in our web app!!
I was thinking to force push on master. Has anyone tried this, will that work?!

**Comment by Hocuri** [comment]
Yup, force push works. You may even want to create an Alias in Git so that every push automatically force pushes.

**Comment by Hocuri** [comment]
What will also help is `export EDITOR="rm -rf"`. This will help you get rid of lots of unnecessary files on your computer, many of which might contain code!
